identity-server/src/main/java/com/hevelian/identity/server/config/rest/RestContextConfig.java

```diff
@@ -85,6 +95,7 @@ public RestHandlerExceptionResolver restExceptionResolver() {
         // with wrong behavior.
         .contentNegotiationManager(contentNegotiationManager)
         .httpMessageConverters(requestMappingHandlerAdapter.getMessageConverters())
+        .messageInterpolator(messageInterpolator())
```

Comment:
Add comment:
//Custom message interpolator to resolve Spring beans in messages.properties

Reply:
Fixed